lib/cartopy/mpl/gridliner.py

```diff
@@ -83,11 +85,27 @@ def _north_south_formatted(latitude, num_format='g'):
 
 
 #: A formatter which turns longitude values into nice longitudes such as 110W
-LONGITUDE_FORMATTER = mticker.FuncFormatter(lambda v, pos:
-                                            _east_west_formatted(v))
+_LONGITUDE_FORMATTER = mticker.FuncFormatter(lambda v, pos:
+                                             _east_west_formatted(v))
 #: A formatter which turns longitude values into nice longitudes such as 45S
-LATITUDE_FORMATTER = mticker.FuncFormatter(lambda v, pos:
-                                           _north_south_formatted(v))
+_LATITUDE_FORMATTER = mticker.FuncFormatter(lambda v, pos:
+                                            _north_south_formatted(v))
+
+
+def deprecated_formatter(old_formatter, old_formatter_name,
+                         new_formatter_name):
+    warnings.warn("{} has been deprecated. Please use {} instead".format(
+        old_formatter_name, new_formatter_name),
+                  DeprecationWarning)
+    return old_formatter
+
+
+LONGITUDE_FORMATTER = deprecated_formatter(
```

Comment:
You are calling the function immediately when the module is imported, so this warning will always be shown regardless of whether the user actually wants to use `LONGITUDE_FORMATTER` or `LATITUDE_FORMATTER`. I think this will create too much noise and not target those people you want to see the warning.